src/components/Main_head.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import '../style/Main-head.css'
+function Main_head() {
```

Comment:
Again, why another component for just a single line?

Reply:
done